src/check/arbitrary/CloneArbitrary.ts

```diff
@@ -55,6 +55,10 @@ class CloneArbitrary<T> extends Arbitrary<T[]> {
       cur = its.map((it) => it.next());
     }
   }
+
+  withBias(freq: number): CloneArbitrary<T> {
+    return new CloneArbitrary(this.arb.withBias(freq), this.numValues);
```

Comment:
In terms of performances using `biasWrapper` may have been a boost, but as we will migrate this arbitrary to `NextArbitrary` it's not needed to change that.